scanner/src/main/kotlin/Scanner.kt

```diff
@@ -216,17 +209,16 @@ class Scanner(
 
         val issues = controller.getIssues()
 
-        val scannerRun = ScannerRun.EMPTY.copy(
+        return ScannerRun.EMPTY.copy(
```

Comment:
As this now sets everything but `environment`, it's a bit cheaper to not `copy()` but construct an instance:

```kotlin
return ScannerRun(
    // ...
    environment = Environment(),
    // ...
)
```
)